background.js

```diff
@@ -59,6 +58,24 @@ function getRandomInt(min, max) {
     return Math.floor(Math.random() * (max - min) + min); 
 }
 
+// get an array that holds selected number of index for contents array
+// to avoid the duplicate of contents
+function getRandomIndexes(max, selectedItemNumber) {
```

Comment:
The naming of this variable is shit because I don't know what the maximum value means.